android/capacitor/src/main/java/com/getcapacitor/plugin/CapacitorCookieManager.java

```diff
@@ -36,18 +39,19 @@ public CapacitorCookieManager(Bridge bridge) {
     public CapacitorCookieManager(CookieStore store, CookiePolicy policy, Bridge bridge) {
         super(store, policy);
         webkitCookieManager = android.webkit.CookieManager.getInstance();
-        this.bridge = bridge;
+        this.localUrl = bridge.getLocalUrl();
+        this.serverUrl = bridge.getServerUrl();
```

Comment:
replace storing the Bridge with storing the strings needed for `getSanitizedDomain`